generators/app/package-json.js

```diff
@@ -250,6 +249,12 @@ const packageJSON = (current, context) => {
   // here because the videojs-languages package will create the destination
   // directory if needed.
   if (context.lang) {
+    if (!context.docs) {
```

Comment:
Another small fix to add `build:lang`, and `build:css` to `pretest`. Since we could need those for tests. js building is already done by karma itself.